<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Search</class>
 <widget class="QDialog" name="Search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1271</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>200</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #e7e7e7;
    border-radius: 13px;
    background-color: #232323;
    color: white;</string>
   </property>
   <property name="title">
    <string>Search Type</string>
   </property>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>110</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0;
font-family: 'Lobster', cursive;
color: rgb(255, 255, 255);
font-weight: 900;</string>
    </property>
    <property name="text">
     <string>Category</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>110</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0;
font-family: 'Lobster', cursive;
color: rgb(255, 255, 255);
font-weight: 900;</string>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="searching">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>160</y>
     <width>441</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #e7e7e7;
    border-radius: 13px;
    background-color: #232323;
    color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>80</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 28pt;
font: 28pt &quot;Snap ITC&quot;;
color: rgb(255, 255, 127);
font-weight: 900;
text-align: center;</string>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>462</x>
     <y>250</y>
     <width>241</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 127);
color: rgb(0, 0, 0);
border-radius:13.5px;
height: 30px;</string>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addproduct">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>450</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #e7e7e7;
    border-radius: 13px;
    background-color: #232323;
    color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="addquantity">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>450</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #e7e7e7;
    border-radius: 13px;
    background-color: #232323;
    color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>450</y>
     <width>381</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 127);
color: rgb(0, 0, 0);
border-radius:13.5px;
height: 30px;</string>
   </property>
   <property name="text">
    <string>Add Product To My Cart</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>310</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #e7e7e7;
    border-radius: 13px;
    background-color: #232323;
    color: white;
border:0;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>450</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Lobster', cursive;
color: rgb(255, 255, 255);
font-weight: 900;</string>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>450</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Lobster', cursive;
color: rgb(255, 255, 255);
font-weight: 900;</string>
   </property>
   <property name="text">
    <string>Quantity:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>310</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Lobster', cursive;
color: rgb(255, 255, 255);
font-weight: 900;</string>
   </property>
   <property name="text">
    <string>Search Result:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>340</y>
     <width>841</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Lobster', cursive;
color: rgb(255, 255, 255);
font-weight: 900;</string>
   </property>
   <property name="text">
    <string>====================================================================================</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>370</y>
     <width>831</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 28pt;
font: 28pt &quot;Snap ITC&quot;;
color: rgb(255, 255, 150);
font-weight: 900;
text-align: center;</string>
   </property>
   <property name="text">
    <string>Adding Product To Your Cart</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>420</y>
     <width>841</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Lobster', cursive;
color: rgb(255, 255, 255);
font-weight: 900;</string>
   </property>
   <property name="text">
    <string>====================================================================================</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: 'Lobster', cursive;
color: rgb(255, 255, 255);
font-weight: 900;</string>
   </property>
   <property name="text">
    <string>Name to Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>530</y>
     <width>381</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 127);
color: rgb(0, 0, 0);
border-radius:13.5px;
height: 30px;</string>
   </property>
   <property name="text">
    <string>Back To Main Window</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>530</y>
     <width>381</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 127);
color: rgb(0, 0, 0);
border-radius:13.5px;
height: 30px;</string>
   </property>
   <property name="text">
    <string>Go To Cashier</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
